src/FileManipulator.php

```diff
@@ -46,7 +46,7 @@ public function createDerivedFiles(Media $media, array $only = [], $onlyIfMissin
         $queuedConversions = $profileCollection->getQueuedConversions($media->collection_name);
 
         if ($queuedConversions->isNotEmpty()) {
-            $this->dispatchQueuedConversions($media, $queuedConversions);
+            $this->dispatchQueuedConversions($media, $queuedConversions, $onlyIfMissing);
```

Comment:
One nitpick: could you rename `onlyIfMissing` to `onlyMissing`?

Afterwards I'll be happy to merge this PR!